### PR TITLE
FRAAS-20684 Add How-To for Installing Forge

### PR DESCRIPTION
**Pull request by rdenson**
story: [FRAAS-20684](https://bugster.forgerock.org/jira/browse/FRAAS-20684)

## Summary
`ARTIFACTORY_API_KEY` is required to install forge. This changeset ensures that we:
1. "$HOME/.fraas" exists
1. `ARTIFACTORY_API_KEY` is set

### CODE REVIEW

File: install_forge.sh [hunk 1]
```diff
@@ -6,31 +6,51 @@
...
+  error "ARTIFACTORY_API_KEY not defined"
+  cat << EOF
+Steps to Fix:
+• Create an Artifactory account:
+  ${BLUE}Log into ${ARTIFACTORY_URL} using Backstage${RESET}
+• Open the profile page:
+  ${BLUE}Go to ${ARTIFACTORY_URL}/repo/webapp/#/profile${RESET}
+• Generate an API Key:
+  ${BLUE}Enter password, click "Unlock", click "Generate"/"Regenerate"${RESET}
+• call this script again with ARTIFACTORY_API_KEY="${YELLOW}<API_KEY>${RESET}"
```
Comment (teostandavid) [inline install_forge.sh:33]: Would be helpful to provide an example here how to provide the variable for the uninitiated:

`ARTIFACTORY_API_KEY="<API_KEY>" bash -c "$(curl -fsSL https://raw.githubusercontent.com/ForgeCloud/public-scripts/master/install_forge.sh)"`

File: install_forge.sh [hunk 2]
```diff
@@ -6,31 +6,51 @@
 #   bash -c "$(curl -fsSL https://raw.githubusercontent.com/ForgeCloud/public-scripts/master/install_forge.sh)"
 #
 # Optional Vars (the defaults will work for most):
-# • ARTIFACTORY_API_KEY : API key for Artifactory auth
 # • FORGE_VERSION       : Version to install
 # • FRAAS_CONFIG_PATH   : Path to the FRaaS config file
```
Comment (teostandavid) [inline install_forge.sh:10]: If we're providing the ARTIFACTORY_API_KEY inline is the config file (FRAAS_CONFIG_PATH) still needed?

Viper config is stored in forge.yaml and the user needs to run init to set it again, not great in itself.

Comment (rdenson) [reply install_forge.sh:10]: this is why it was removed:
```sh
# Optional Vars (the defaults will work for most):
# • ARTIFACTORY_API_KEY : API key for Artifactory auth
```
`ARTIFACTORY_API_KEY` envar is not optional

File: install_forge.sh (hunk 1 above)
Comment (rdenson) [reply install_forge.sh:33]: Yes sir, that's the idea albeit not here. We want the public scripts here (_I suppose for now_) and then in our docs (gh + stash) we'll have the "how-to".

File: install_forge.sh (hunk 2 above)
Comment (rdenson) [reply install_forge.sh:10]: >_Viper config is stored in forge.yaml and the user needs to run init to set it again, not great in itself._

this thing... so in another PR (_in another repo_) that flow is fixed

File: install_forge.sh (hunk 1 above)
Comment (teostandavid) [reply install_forge.sh:33]: I meant more the syntax of providing the ENV variable in the call to action in the script. Instructions are nice when they're straightforward.

**Review by TeodorStandavid — APPROVED** [review]

**Pull request merged**